datafusion/sql/src/expr/mod.rs

```diff
@@ -478,7 +492,6 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 trim_where,
                 trim_what,
                 trim_characters,
-                ..
```

Comment:
turns out there are no other fields. By removing `..` it means if any new fields are added in future sqlparser releases they will need to be handled explicitly